packages/astro/src/assets/fonts/types.ts

```diff
@@ -0,0 +1,27 @@
+import type { BUILTIN_PROVIDERS } from './constants.js';
+import type { GOOGLE_PROVIDER_NAME } from './providers/google.js';
+import type { LOCAL_PROVIDER_NAME } from './providers/local.js';
+
+export interface FontProvider<TName extends string> {
+	name: TName;
+	entrypoint: string;
```

Comment:
`entrypoint` should be a string or a `URL`. Since v5, all entry points now accept `URL` too

Reply:
I'll address this in the next PR